blog/_posts/2018-07-09-dvteam-activated.md

```diff
@@ -0,0 +1,75 @@
+---
+title: DV Team Activated!
+layout: post
+created_at: Mon Jul 09 2018
+permalink: blog/dvteam_activated

```

Comment:
Hi @artnerdnet ! Thanks for the PR. Please check the comments below.

Permalink should have same file name as your markdown file [along with date]. You can check the other posts for example.